k8s/k8s.go

```diff
@@ -644,6 +653,28 @@ func getK8sLabelsAnnotations(client *kubernetes.Clientset, k8sargs utils.K8sArgs
 	return labels, pod.Annotations, nil
 }
 
+func getK8sPodPorts(client *kubernetes.Clientset, k8sargs utils.K8sArgs) ([]api.EndpointPort, error) {
+	var c k8sbackend.Converter
+	var bc backendConverter.WorkloadEndpointConverter
+
+	pod, err := client.Pods(string(k8sargs.K8S_POD_NAMESPACE)).Get(fmt.Sprintf("%s", k8sargs.K8S_POD_NAME), metav1.GetOptions{})
```

Comment:
Hm, this means we're now doing two queries for the same information in the k8s API.

I think we should combine this with the get labels + annotations function so that we only need to query for the pod once.